evm/src/cpu/kernel/asm/main.asm

```diff
@@ -13,6 +13,11 @@ global main:
 
     // Initialise the shift table
     %shift_table_init
+    // Encode constant nodes
+    %initialize_rlp_segment
+
+    // Encode constant nodes
+    %initialize_rlp_segment
```

Comment:
Is it a duplicate? If not we may want extra documentation, this is a bit confusing.